src/features/auth/ui/FindPasswordForm.tsx

```
@@ -1,56 +1,77 @@
-                                    value={findEmail}
-                                    onChange={handleChangeFindPassword}
-                                />
-                            </div>
-                            {errorMessage && <div className='flex items-center gap-[4px]'><Image src="/icons/alert.svg" alt="error" width={10} height={10} priority className='mb-[2px]' /><span className='text-[12px] font-[700] text-[#EC3030]'>{errorMessage}</span></div>}
-                            {successMessage && <div className='flex items-center gap-[4px]'><span className='text-[10px] font-[700] text-[#00E35B]'>{successMessage}</span></div>}
-                            <button className='h-[37px] border-[1px] border-[#DEDEDE] rounded-[7px] bg-[#969696] text-white text-[15px] font-[800]' onClick={handleFindPasswordClick}>인증 메일 보내기</button>
-                        </div>
-                    </div>
+                  <label className="block text-sm font-medium text-gray-300 mb-2">이메일</label>
+                  <div className="relative">
+                    <Mail
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)